core/startos/src/lib.rs

```diff
@@ -222,19 +247,22 @@ pub fn server<C: Context>() -> ParentHandler<C> {
             .with_custom_display_fn(|_handle, result| {
                 Ok(firmware::display_firmware_update_result(result))
             })
-            .with_call_remote::<CliContext>(),
+            .with_call_remote::<CliContext>()
+            .with_about("Check for firmware updates and update if available"),
```

Comment:
"Update the mainboards firmware to the latest firmware available in this version of StartOS if available. Note: This command does not reach out to the Internet"